### PR TITLE
py-unidecode: new package at 1.1.1

### PR DESCRIPTION
**Pull request by codeandkey**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-unidecode/package.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+class PyUnidecode(PythonPackage):
+    """ASCII transliterations of Unicode text"""
+
+    homepage = "https://pypi.org/project/Unidecode"
+    url      = "https://pypi.io/packages/source/u/unidecode/Unidecode-1.1.1.tar.gz"
+
+    version('1.1.1', sha256='2b6aab710c2a1647e928e36d69c21e76b453cd455f4e2621000e54b2a9b8cce8')
+    version('0.04.21', sha256='280a6ab88e1f2eb5af79edff450021a0d3f0448952847cd79677e55e58bad051')
+
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-unidecode/package.py:17]: Missing:
```python
depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
```

File: var/spack/repos/builtin/packages/py-unidecode/package.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    depends_on('py-setuptools', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-unidecode/package.py:18]:
```suggestion
    depends_on('py-setuptools', type='build')
```

File: var/spack/repos/builtin/packages/py-unidecode/package.py (hunk 1 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-unidecode/package.py:17]: added

File: var/spack/repos/builtin/packages/py-unidecode/package.py (hunk 2 above)
Comment (codeandkey) [reply var/spack/repos/builtin/packages/py-unidecode/package.py]: updated!

**Pull request merged**